resources/monitoring/templates/prometheus/servicemonitor.yaml

```diff
@@ -2,26 +2,22 @@
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  name: {{ template "prometheus-operator.fullname" . }}-prometheus
-  namespace: {{ $.Release.Namespace }}
+  name: {{ template "kube-prometheus-stack.fullname" . }}-prometheus
+  namespace: {{ template "kube-prometheus-stack.namespace" . }}
   labels:
-    app: {{ template "prometheus-operator.name" . }}-prometheus
-{{ include "prometheus-operator.labels" . | indent 4 }}
+    app: {{ template "kube-prometheus-stack.name" . }}-prometheus
+{{ include "kube-prometheus-stack.labels" . | indent 4 }}
 spec:
   selector:
     matchLabels:
-      app: {{ template "prometheus-operator.name" . }}-prometheus
+      app: {{ template "kube-prometheus-stack.name" . }}-prometheus
       release: {{ $.Release.Name | quote }}
       self-monitor: "true"
   namespaceSelector:
     matchNames:
-      - {{ $.Release.Namespace | quote }}
+      - {{ printf "%s" (include "kube-prometheus-stack.namespace" .) | quote }}
   endpoints:
   - port: {{ .Values.prometheus.prometheusSpec.portName }}
-    metricRelabelings:
```

Comment:
We should keep these.

Reply:
These are important to keep the prometheus memory usage under control

Reply:
Good catch, I moved them to the value files to keep the `ServiceMonitor` definition unchanged